language/vm/src/proptest_types.rs

```diff
@@ -94,11 +92,11 @@ impl CompiledModuleStrategyGen {
         Self {
             size: size as usize,
             field_count: (0..5).into(),
-            struct_type_params: (0..2).into(),
+            struct_type_params: (0..3).into(),
```

Comment:
this is the number of type args that are small for now though I don't think it's a bad thing.
Basically it will create generic up to Cup<T, W>.
This is also a side effect of the simple strategies we use for proptest. We are effectively asking for a random number between 0 and 2 and that makes the chances to get a `Struct` (which is 0) pretty good as opposed to a wider range. And getting a decent number of `Struct` as opposed to `StructInstantiation` makes things simpler and converging faster